bin/run_pypit

```diff
@@ -83,11 +88,11 @@ def main() :
 
     # Execute the reduction, and catch any bugs for printout
     if debug['develop']:
-        pypit.PYPIT(red, progname=pypit.__file__, quick=qck, ncpus=cpu, verbose=vrb,
+        pypit.PYPIT(red, initmsgs, progname=pypit.__file__, quick=qck, ncpus=cpu, verbose=vrb,
```

Comment:
As I comment below in PYPIT, I think we should not pass initmsgs here. Instead, we should pass a develop keyword (like what is presently done with the verbose keyword), or perhaps better, a dictionary containing some of the relevant bits of information for the messages (including `verbose, develop, last_updated, version`)